cli/features/tutorial.feature

```diff
@@ -36,9 +36,9 @@ Feature: Following the tutorial
       description: A todo application
       version: 0.0.1
 
-      bus:
-        type: exocom
-        version: 0.21.7
+      dependencies:
+        - type: exocom
+          version: 0.21.7
```

Comment:
Sticking with `type` and the name is going to be the type and version concatenated